src/scss/components/checkbox/_index.scss

```diff
@@ -125,17 +126,22 @@ hx-checkbox-control {
 
     &:checked,
     &:indeterminate {
+      color: $blue-700;
       color: var(--hxCheckbox-indeterminate-color, $blue-700);
 
       ~ label > hx-checkbox {
+        background-color: $blue-700;
+        background-color: var(--hxCheckbox-indeterminate-label-bgcolor, $blue-700);
+        border-color: $blue-700;
         border-color: var(--hxCheckbox-indeterminate-label-bordercolor, $blue-700);
-        color: var(--hxCheckbox-indeterminate-label-color, $blue-700);
+        color: $gray-0;
+        color: var(--hxCheckbox-indeterminate-label-color, $gray-0);
       }
     }
 
     &:valid {
       ~ label > hx-checkbox:hover {
-        @include hxCheckboxFacade($pseudo-state: hover);
+        @include hxCheckboxFacade($pseudo-state: null);  // v2.0 specs
```

Comment:
we didn't still need the hover here? I guess because hover technically is supposed to look the same as the checked state, correct?

Reply:
Yes, we need clarification from design on `hover` while the checkbox is `checked`/`selected` state.

Reply:
@danielmdesigns, can you please provide design recommendations for `hover` state on a `checked`/`selected` checkbox.

Reply:
@ssalinas24, I'm leaving the hover state construct in place for now.